handout/routing/config.md

```diff
@@ -49,6 +51,13 @@ const routes: Routes = [
 ];
 
 export const routing = RouterModule.forRoot(routes);
+```
+
+_app/index.ts_
```

Comment:
Yep I was about to do that but the later plunker examples use `index.ts`. I'll update each of the plunkers to use `app.module.ts` instead.